ingestionpolicies_example_test.go

```diff
@@ -22,33 +22,34 @@ func ExampleIngestionPolicies() {
 
 	ingestionPolicies := client.IngestionPolicies()
 
-	policy := &wavefront.IngestionPolicy{
+	policyRequest := &wavefront.IngestionPolicyRequest{
 		Name:        "test ingestion policy",
 		Description: "an ingestion policy created by the Go SDK test suite",
 		Scope:       "ACCOUNT",
+		Accounts:    []string{"user@example.com"},
 	}
 
-	err = ingestionPolicies.Create(policy)
+	ingestionPolicy, err := ingestionPolicies.Create(policyRequest)
 
 	if err != nil {
 		log.Fatal(err)
 	}
 
 	// The ID field is now set, so we can update/delete the policy
-	fmt.Println("policy ID is", policy.ID)
+	fmt.Println("policy ID is", ingestionPolicy.ID)
 
 	// Change the description
-	policy.Description = "an ingestion policy updated by the Go SDK test suite"
+	policyRequest.Description = "an ingestion policy updated by the Go SDK test suite"
```

Comment:
I like the idea of doing a find here. I made a few tests where I do a find and then update the found object. Could be good for the example too.

I also like the idea of a method that uses a response object directly to do work. (To be honest I was struggling to learn the framework here and get the request and response object types all plugged in and working so I kind of gave up once I had something that seemed reasonably tidy 😅. Kind of an MVP that would at least get the provider fixed. (I also noticed a lot of these objects seem to be missing methods. 🤔))

Reply:
What does MVP stand for in this context? Yea, converting response objects to request objects internally is a good idea, but there are tradeoffs because the response objects are often more complicated to fill out than the requests objects. I don't have a strong preference either way.

Reply:
I am knowingly using MVP pretty vaguely here (I think as a term it is highly subjective). In this context MVP in my mind is basically "functional" if a bit rough around the edges.

Cause ultimately, I don't think this resource is actually used very much and my ultimate goal here is to fix the terraform provider not the CLI and having a slightly clunky CLI isn't really a blocker for the fixing-the-provider goal.